Documentation/ApiOverview/Context/_MyController.php

```diff
@@ -9,6 +9,6 @@
 final class MyController
 {
     public function __construct(
-        private readonly Context $context,
+        private readonly Context $context
```

Comment:
You don't have to delete the comma, it is valid PHP code since some versions.

Reply:
Yes, but I prefer it without a trailing comma.

Reply:
This change has nothing to do with the PR you propose. It also doesn't throw an error. So this change is superfluous.

Reply:
I did not want to make an extra PR for just one comma.

Reply:
It is not wrong and because you prefer the other way personally, that does not justify the change. Instead there is a good reason for this: Diffs are better readable when another constructor argument is added as only the new line is highlightes.

Reply:
I do not mean to push my preferences. This is a misunderstanding!
I have just never seen this yet anywhere in the TYPO3 Documentation.

Reply:
I have restored the superfluous comma at the end.